contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -243,6 +248,7 @@ func (s *SchedulerServer) addCoreFlags(fs *pflag.FlagSet) {
 	fs.Var(&s.DefaultContainerMemLimit, "default-container-mem-limit", "Containers without a memory resource limit are admitted this much amount of memory in MB")
 	fs.BoolVar(&s.ContainPodResources, "contain-pod-resources", s.ContainPodResources, "Reparent pod containers into mesos cgroups; disable if you're having strange mesos/docker/systemd interactions.")
 	fs.BoolVar(&s.AccountForPodResources, "account-for-pod-resources", s.AccountForPodResources, "Allocate pod CPU and memory resources from offers (Default: true)")
+	fs.DurationVar(&s.nodeRelistPeriod, "node-relist-period", s.nodeRelistPeriod, "Period between relisting of all nodes from the apiserver.")
```

Comment:
I think I commented directly on the commit earlier, but I'd like to see us reuse the `node-monitor-period` flag name since it means the same thing